packages/instrumentation-typeorm/src/utils/skip-instrumentation.ts

```diff
@@ -0,0 +1,6 @@
+import { trace, context } from '@opentelemetry/api';
+import { TypeormInstrumentationConfig } from '../types';
+
+export function shouldSkipInstrumentation(config: TypeormInstrumentationConfig) {
+    return config.requireParentSpan === true && trace.getSpan(context.active()) === undefined;
```

Comment:
`return config.requireParentSpan && trace.getSpan(context.active()) === undefined;` should work, there is no need for the `=== true` check